guaymas_missions/sentry_rosette_transect/transect_data_parsing.py

```diff
@@ -109,22 +110,34 @@
     bott_df["bottle_pos"] = bott_df["bottle_pos"].astype(np.int32)
     gga_df = pd.read_table(INPUT_GGA, delimiter=",")
     gga_df["CTD Bottle #"] = gga_df["CTD Bottle #"].astype(np.int32)
+    
     gga_df.loc[:, 'ch4_ppm_corr_05'] = gga_df["GGA Methane"] / 0.05
     gga_df.loc[:, 'ch4_ppm_corr_15'] = gga_df["GGA Methane"] / 0.15
+    gga_df.loc[:, 'ch4_ppm_corr_12'] = gga_df["GGA Methane"] / 0.12
```

Comment:
Commenting the source and reason for these corrections would help us in the future; probably worth doing

Reply:
Good idea! Added!